SOCKET-EVENTS.md

```diff
@@ -1,12 +0,0 @@
-+-----------------+------------------------------------------------------------------------------+----------+-----------+-------------+---------------------------------------------------------------------------+
```

Comment:
We may still want to document these socket events somewhere as they are still considered an interface